scripts/eurostat/process_eurostat.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+import argparse
+import gzip
+import shutil
+
+
+def unzip_single(filename_in):
+    filename_out = filename_in[:-3]  # to trim the .gz
+    with gzip.open(filename_in, 'rb') as f_in:
+        with open(filename_out, 'wb') as f_out:
+            shutil.copyfileobj(f_in, f_out)
+    return filename_out
+
+
+def first_stage_processing(df, separation_symbol):
```

Comment:
Would a more descriptive function name help the reader?  Unfortunately, I am not sure what to suggest.